src/dowel/tensor_board_output.py

```diff
@@ -72,52 +73,57 @@ def __init__(self,
     @property
     def types_accepted(self):
         """Return the types that the logger may pass to this output."""
+        types = (plt.Figure, scipy.stats._distn_infrastructure.rv_frozen,
+                 scipy.stats._multivariate.multi_rv_frozen,
+                 Histogram) + np.ScalarType
+        types = list(types)
+        types.remove(str)
```

Comment:
what is going on here? why can't this just be a tuple? how is `str` even in this tuple `types` when you *just* defined it as a literal (not including str) 2 lines above? if you did find a `str` in there for some reason, how did it get there?